lib/model/Capability.js

```diff
@@ -4,19 +4,35 @@ const Color = require('color');
 const Range = require(path.join(__dirname, 'Range.js'));
 
 module.exports = class Capability {
-  constructor(jsonObject) {
+  constructor(jsonObject, channel) {
     this.jsonObject = jsonObject; // calls the setter
+    this._channel = channel;
   }
 
   set jsonObject(jsonObject) {
     this._jsonObject = jsonObject;
+    this._cache = {};
   }
 
 
   get range() {
     return new Range(this._jsonObject.range); // required
   }
 
+  get ranges() {
+    if (!('ranges' in this._cache)) {
+      this._cache.ranges = [];
+      for (let i = this._channel.maxFineness; i >= 0; i--) {
```

Comment:
This is intended, as a fineness of 1 means this is a 16bit channel and this getter should return the range in all finenesses (8bit and 16bit in this case).

However, you're right in that I need a better name and a describing comment